packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -272,6 +272,17 @@ export default class VTable {
         }
     }
 
+    /**
+     * Loop each cell of the table and invoke a callback function
+     * @param callback The callback function to invoke
+     */
+    forEachCell(callback: (cell: VCell) => any) {

```

Comment:
I prefer not adding this public method unless it will be used by multiple files. You can move it into TableResize.ts. So that we don't need to bump minor version number.

Reply:
fixed